mava/configs/network/qmix_rnn.yaml

```diff
@@ -0,0 +1,19 @@
+# ---Recurrent Structure Networks---
+hidden_state_dim: 256 # The size of the RNN hiddenstate for each agent.
+
+q_network:
+  pre_torso:
```

Comment:
I don't like that because you end up with q_network configs in PPO systems and vice versa. I actually think we should have an rnn_actor_critic.yaml, rnn_qlearning.yaml and rnn_qmix.yaml. But feel we can do this in another PR.

Reply:
Totally agree!